tests/test_types_structure.py

```diff
@@ -801,6 +801,39 @@ def __init__(self, _0 = None, _1 = None, _2 = None, _3 = None, _4 = None):
     assert result is cached
 
 
+def test_structure_definition_newline(cs: cstruct, compiled: bool) -> None:
+    cdef = """
+    struct test {
+        char    magic[4
+        ];
+
+        wchar   wmagic[4
+        ];
+        uint8   a;
+        uint16  b;
+        uint32  c;
+        char    string[];
+        wchar   wstring[];
+    };
+    """
+    cs.endian = ">"
+    cs.load(cdef, compiled=compiled)
+
+    assert verify_compiled(cs.test, compiled)
+
+    buf = b"test\x00t\x00e\x00s\x00t\x01\x02\x03\x04\x05\x06\x07lalala\x00\x00t\x00e\x00s\x00t\x00\x00"
+
+    obj = cs.test()
+    obj.magic = "test"
+    obj.wmagic = "test"
+    obj.a = 0x01
+    obj.b = 0x0203
+    obj.c = 0x04050607
+    obj.string = b"lalala"
+    obj.wstring = "test"
+
+    assert obj.dumps() == buf
+
```

Comment:
```suggestion


```
I'm not sure how this passes the linter, but I think this should have two newlines.

I think line 279 is wrong too, can you maybe also add a newline there?